AST-739 - Offcanvas header's preview is displaying on Legacy widget's preview in customizer of WordPress 5.8. (#3069)
brainstormforce/astra: changelog.txt

v3.6.8 (Unreleased)
- Improvement: Added WPML translation support for all customizer's strings through wpml-config.xml file.
- Fix: Bulk action selection not working on any WordPress list table when theme's "Get Started" notice is active.

> - Fix: Header Footer Widgets: Offcanvas header's is displaying over Legacy widget's preview in customizer of WordPress 5.8.

Reviewer: ```suggestion
- Fix: Header Footer Widgets: Off-canvas content is displaying over Legacy widget's preview in the customizer for WordPress 5.8.
```
